bundler.d/nulldb.rb

```diff
@@ -1 +1,4 @@
+group :nulldb do
+  gem 'activerecord-nulldb-adapter'
+end
 gem 'activerecord-nulldb-adapter'
```

Comment:
Don't we have this twice now?

Reply:
We do, but: the group one is needed for our packaging tooling, and the non-group one needed for nulldb to properly initialize, see the discussion ewoud linked above

Reply:
Ok, I believe this is one of the rare cases when a comment in the code is in order.